src/ondisk.rs

```diff
@@ -11289,6 +11300,11 @@ make_token_accessors! {
         #[cfg_attr(feature = "serde-hex", serde(serialize_with = "SerHex::<StrictPfx>::serialize", deserialize_with = "SerHex::<StrictPfx>::deserialize"))]
         DimmSensorCritical(default 95, id 0x38e9bf5d) | pub get u16 : pub set u16, // Rome (Obsolete)
 
+        #[cfg_attr(feature = "serde-hex", serde(serialize_with = "SerHex::<StrictPfx>::serialize", deserialize_with = "SerHex::<StrictPfx>::deserialize"))]
+        LrdimmSensorUpper(default 66, id 0xcddd55d0) | pub get u16 : pub set u16, // Milan 1.0.0.g
+        #[cfg_attr(feature = "serde-hex", serde(serialize_with = "SerHex::<StrictPfx>::serialize", deserialize_with = "SerHex::<StrictPfx>::deserialize"))]
+        LrdimmSensorCritical(default 80, id 0x22000b2b) | pub get u16 : pub set u16, // Milan 1.0.0.g
```

Comment:
Looks like they've been defined in agesa since 1.0.0.b at least.

Reply:
You're right! Thanks. Fixed.